<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Uzzal_Das" failures="0" tests="3" name="Test(failed)" time="0.051" errors="0" timestamp="2023-05-15T13:28:13 EDT">
  <testcase classname="framework.testng.test.RunFailedTestCase" name="getLogin" time="0.009"/>
  <testcase classname="framework.testng.test.RunFailedTestCase" name="getLogout" time="0.001"/>
  <testcase classname="framework.testng.test.RunFailedTestCase" name="getUrl" time="0.001"/>
  <testcase classname="framework.testng.test.RunFailedTestCase" name="getPass" time="0.0">
    <ignored/>
  </testcase> <!-- getPass -->
  <testcase classname="framework.testng.test.RunFailedTestCase" name="getUser" time="0.0">
    <ignored/>
  </testcase> <!-- getUser -->
</testsuite> <!-- Test(failed) -->
